sale_input_barcode_gs1/tests/test_sale_input_barcode_gs1.py

```diff
@@ -4,7 +4,7 @@
 from odoo.tests import TransactionCase
 
 
-class Test(TransactionCase):
+class TestSaleInputBarcodeGs1(TransactionCase):
```

Comment:
Not blocking comment.

In the log, what is the more readable.

BEFORE
`INFO odoo odoo.addons.sale_input_barcodes_gs1.tests.test_module: Starting Test.test_my_test`
AFTER
`INFO odoo odoo.addons.sale_input_barcodes_gs1.tests.test_sale_input_barcode_gs1: Starting TestSaleInputBarcodeGs1.test_my_test`

DRY : don't repeat yourself ;-)

I think first is better, probably my taste.

The last one consume more chars, more hard disk space ;-)